src/components/Lesson/Lesson.js

```diff
@@ -55,6 +55,28 @@ const onclickAndSetCheckboxes = (path, checkboxes, setCheckbox) => {
   }
 };
 
+const resetLesson = (path, setCheckbox) => {
+  const labels = document.getElementsByTagName('label');
+  for (let label of labels) {
+    const input = document.getElementById(label.htmlFor);
+    if (input && input.type === 'checkbox') {
+      let hash = hashCode(label.textContent);
+      setCheckbox(path, hash, false);
+      location.reload();
+    }
+  }
+};
+
+const checkboxTrue = (checkboxes) => {
+  let counter = 0;
+  for (let i in checkboxes) {
+    if (checkboxes[i] === true) {
+      counter++;
+    }
+  }
+  return counter === 0 ? false : true;
```

Comment:
Hele `checkboxTrue` kan gjøres enklere, og jeg foreslår at du omdøper den til `anyCheckboxTrue`.
Her er forslaget mitt:
```
const anyCheckboxTrue = (checkboxes) => {
  for (let i of Object.keys(checkboxes)) {
    if (checkboxes[i] === true) {
      return true;
    }
  }
  return false;
};
```
(Grunnen til at jeg bruker `Object.keys` er for å unngå å få med evt keys fra prototypen til checkboxes. Alternativt kunne man brukt `for ... in` slik du gjorde, men legge til en if-check med `hasOwnProperty`)